packages/interactor/test/define-interactor.test.ts

```diff
@@ -0,0 +1,220 @@
+import { describe, it } from 'mocha';
+import * as expect from 'expect'
+import { JSDOM } from 'jsdom';
+
+import { defineInteractor, setDefaultOptions } from '../src/index';
+
+process.on('unhandledRejection', () => {
```

Comment:
Is this necessary? I almost always find that when I'm getting `unhandledRejection` warnings, its because I probably should be explicitly catching something (even if I'm going to be ignoring the exception). In other words, explicit error swallowing is almost always preferable to implicit error swallowing.

Reply:
It actually seems to work perfectly fine without it, I can't even remember why I added this 🤔